src/backend/marsha/core/models/video.py

```diff
@@ -628,6 +631,17 @@ def update_reminders(self, step):
             self.reminders = [step]
         self.save()
 
+    def get_generate_salted_hmac(self):
```

Comment:
```suggestion
    def get_generated_salted_hmac(self):
```

And I'm really fan to have a `getter` to get something that is not in the model and also not fan to have logic into the model. But django is model centric so I don't think there is lot of simple alternative.